src/backend/src/loading-app-mux.jl

```diff
@@ -0,0 +1,145 @@
+const API_VERSION = "0.0.1"
+const IN_PRODUCTION = false
+
+PROJECTS_DIR = "projects"
+
+using CSV
+using DataFrames
+
+
+function upload_file(req) #(project, file, uploadtype, filename)
+    ## TODO nightmare getting multipart form parsing working
```

Comment:
```julia
import HTTP
# Lightly adapted from HTTP.parse_multipart_form
function parse_multipart_form(muxreq::Dict)
    # parse boundary from Content-Type
    m = match(r"multipart/form-data; boundary=(.*)$", first(Iterators.filter(p -> p.first == "Content-Type", muxreq[:headers])).second)
    m === nothing && return nothing

    boundary_delimiter = m[1]

    # [RFC2046 5.1.1](https://tools.ietf.org/html/rfc2046#section-5.1.1)
    length(boundary_delimiter) > 70 && error("boundary delimiter must not be greater than 70 characters")

    return HTTP.parse_multipart_body(muxreq[:data], boundary_delimiter)
end

# Get stringified version of form payload from a request
const formreq2data(req) = String(read(parse_multipart_form(req)[1].data))
```

Quick and dirty solution. A neater solution would be to tell Julia how to convert between Mux style requests and HTTP.jl style requests, but this was quicker : )